README.md

```diff
@@ -1,5 +1,6 @@
 # Rechnung
 
+[![Django CI](https://github.com/Segelzwerg/Rechnung/actions/workflows/django.yml/badge.svg)](https://github.com/Segelzwerg/Rechnung/actions/workflows/django.yml)
```

Comment:
```suggestion
[![CI](https://github.com/Segelzwerg/Rechnung/actions/workflows/django.yml/badge.svg)](https://github.com/Segelzwerg/Rechnung/actions/workflows/django.yml)
```

Reply:
Does make a difference in the Badge name. I'll already tried.

Reply:
Then change the workflow name please